packages/notify-client/src/controllers/engine.ts

```diff
@@ -61,10 +62,17 @@ export class NotifyEngine extends INotifyEngine {
 
   public register: INotifyEngine["register"] = async ({
     account,
+    isLimited,
     onSign,
     domain,
   }) => {
-    const statement = NOTIFY_AUTHORIZATION_STATEMENT;
+    // explicity check if it was set to false because null/undefined should count as
```

Comment:
Nice context for the non-obvious explicit false check 👌 

```suggestion
    // Explicitly check if it was set to false because null/undefined should count as
```